utils/fix_rules.py

```diff
@@ -257,6 +286,77 @@ def rewrite_value_remove_prefix(line):
     return key + ": " + new_value
 
 
+class CCEFile:
```

Comment:
@matejak Does it make sense to move this to say, `ssg/cce.py` and move the `is_cce_format_valid` and `is_cce_value_valid` into there as well (from `ssg/checks.py`)?